README.md

```diff
@@ -39,98 +41,165 @@ Current stable release is able to persist Orion context data in:
-                  }
+        "entities": [
+            {
+                "type": "car",
+                "isPattern": "false",
+                "id": "car1"
+            }
+        ],
+        "attributes": [
+            "speed",
+            "oil_level"
+        ],
+        "reference": "http://localhost:5050/notify",
+        "duration": "P1M",
+        "notifyConditions": [
+            {
+                "type": "ONCHANGE",
+                "condValues": [
+                    "speed"
                 ]
-              }
-            ],
-            "type" : "Room",
-            "isPattern" : "false",
-            "id" : "Room1"
-          },
-          "statusCode" : {
-            "code" : "200",
-            "reasonPhrase" : "OK"
-          }
-        }
-      ]
+            }
+        ],
+        "throttling": "PT1S"
     }
+    EOF
+
+Which means: <i>Each time the the 'car1' entity, of type 'car', changes its value of 'speed' send a notification to http://localhost:5050/notify (where Cygnus will be listening) with the 'speed' and 'oil_level' values. This subscription will have a duration of one month, and please, do not send me notifications more than one per second</i>.
+
+[Top](#top)
 
-Such a notification is sent by Orion to the default Flume HTTP source, which relies on the developed OrionRestHandler for checking its validity (that it is a POST request, that the target is 'notify' and that the headers are OK), detecting the content type (that it is in Json format), extracting the data (the Json part) and finally creating a Flume event to be put in the channel:
+###<a name="section3.2"></a>NGSI notification reception
+Let's supose the 'speed' of the 'car1' entity changes to '112.9'; then the following NGSI notification (or NGSI event) would be sent as a Http POST to the configured Cygnus listener, i.e. the native `HTTPSource` (the code below is an <i>object representation</i>, not any real data format):
+
+    ngsi-event={
+        http-headers={
+            Content-Length: 492
+            User-Agent: orion/0.9.0
+            Host: localhost:1028
+            Accept: application/xml, application/json
+            Content-Type: application/json
+            Fiware-Service: vehicles
+            Fiware-ServicePath: 4wheels 
+        },
+        payload={
+            {
+                "subscriptionId" : "51c0ac9ed714fb3b37d7d5a8",
+                "originator" : "localhost",
+                "contextResponses" : [
+                    {
+                        "contextElement" : {
+                        "attributes" : [
+                            {
+                                "name" : "speed",
+                                "type" : "kmh",
+                                "value" : "112.9",
+                                "metadatas": []
+                            },
+                            {
+                                "name" : "oil_level",
+                                "type" : "percentage",
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/5b6a45eb7c12d4db2e56ac0147eb6851b51c3d9